docs/network-policy.md

```diff
@@ -1,4 +1,39 @@
-# Cluster Network Policy
+# Native Network Policies
+
+## Tier
+
+Antrea supports grouping native NetworkPolicies together in a tiered fashion
+to provide a hierarchy of security policies. This is achieved by setting the
+native NetworkPolicies, like a ClusterNetworkPolicy, with the name of the
```

Comment:
When I first read this, `native NetworkPolicies` sounds like referring to k8s NetworkPolicy. Maybe Antrea native  NetworkPolicies?

Reply:
yeah i used to prefix them with Antrea as well, though @McCodeman mentioned that the project is already Antrea, no need to double re-emphasize on Antrea and he suggested native.. as in native to Antrea.. but if it does causes confusion then i guess we will have to think of alternative way to reference.. like Antrea NetworkPolicies, or NetworkPolicy CRDs .. or something else?

Reply:
I see Calico is also calling its NetworkPolicy CRD as `Calico Network Policy` https://docs.projectcalico.org/security/calico-network-policy, so I suppose Antrea NetworkPolicies should be more appropriate? Want to get others' input on this.

Reply:
damn you Dyangg :)  .. when i think of "native xyz".. i think native to the current project.. i think in the past to refer items native to k8s we mentioned *k8s native* or *native to k8s*.. but since there is confusion .. i will update this to Antrea NetworkPolicies